src/simulation/simulation_transient_voltage_graph.ts

```diff
@@ -23,13 +24,13 @@ export const simulation_transient_voltage_graph = z
       "simulation_transient_voltage_graph",
     ),
     simulation_experiment_id: z.string(),
-    timestamps_ms: z.array(z.number()).optional(),
```

Comment:
Likewise we can keep the original, its confusing but i cant think of a better name